app/src/main/java/org/p2p/wallet/swap/ui/jupiter/settings/presenter/JupiterSwapSettingsPresenter.kt

```diff
@@ -100,9 +101,27 @@ class JupiterSwapSettingsPresenter(
                     view?.bindSettingsList(contentList)
                 }
             }
+            is SwapSettingsPayload -> onDetailsClick(payload)
         }
     }
 
+    private fun onDetailsClick(settingsPayload: SwapSettingsPayload) {
+        when (settingsPayload) {
+            SwapSettingsPayload.ROUTE ->
+                if (canOpenDetails()) view?.showRouteDialog()
+            SwapSettingsPayload.NETWORK_FEE -> view?.showDetailsDialog(SwapInfoType.NETWORK_FEE)
```

Comment:
better to have one style of wrapping when cases 
everything like 
```
-> 
    ....
```
or
`-> ...`